### DISCUSSION TITLE
用的2.2.3版本，导入xcharts时会报错，好像是#if这里失效了，请问是哪里出了问题

### DISCUSSION THREAD

**Discussion by stallion258** [XCharts-Team/XCharts#234] [Q&A] [answered]
![ZX) 4UA%9DACB%~T GFH5QX](https://user-images.githubusercontent.com/55772809/199421984-9a3adf31-4285-46bf-81bc-cd097c897007.png)
![REHEWO4KC~$9YR$8B$ _6P](https://user-images.githubusercontent.com/55772809/199422099-e8fae088-9005-431a-91d0-6da138d8f54b.png)

**Comment by monitor1394** [comment, marked as answer]
这种情况是开启了TextMeshPro了，先确保项目有TextMeshPro插件。
然后参考[issue 194 ](https://github.com/XCharts-Team/XCharts/issues/194)
